quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -68,19 +70,38 @@ void testPropertyEnabledAtRuntime(CLIResult cliResult) {
 
     @Test
     @Launch({"-Dquarkus.log.handler.console.\"console-2\".enable=false", "start", "--http-enabled=true", "--hostname-strict=false"})
+    @DisabledOnOs(value = { OS.WINDOWS })
     @Order(3)
     void testIgnoreQuarkusSystemPropertiesAtStart(CLIResult cliResult) {
         cliResult.assertMessage("Keycloak is the best");
     }
 
+    @Test
+    @Launch({"-Dquarkus.log.handler.console.\\\"console-2\\\".enable=false", "start", "--http-enabled=true", "--hostname-strict=false"})
+    @DisabledOnOs(value = { OS.LINUX, OS.MAC }, disabledReason = "Different handling of quotes within arguments on Windows")
+    @Order(3)
+    void testIgnoreQuarkusSystemPropertiesAtStartWin(CLIResult cliResult) {
+        cliResult.assertMessage("Keycloak is the best");
+    }
+
     @Test
     @Launch({"-Dquarkus.log.handler.console.\"console-2\".enable=false", "build"})
+    @DisabledOnOs(value = { OS.WINDOWS })
     @Order(4)
     void testIgnoreQuarkusSystemPropertyAtBuild(CLIResult cliResult) {
         cliResult.assertMessage("Keycloak is the best");
         cliResult.assertBuild();
     }
 
+    @Test
+    @Launch({"-Dquarkus.log.handler.console.\\\"console-2\\\".enable=false", "build"})
+    @DisabledOnOs(value = { OS.LINUX, OS.MAC }, disabledReason = "Different handling of quotes within arguments on Windows")
```

Comment:
Could this, and the other similar ones, just be EnabledOnOs instead?

Reply:
Yes, both ways work. Intuitively, I also used EnabledOnOs at first, but then I realized that historically we've tried to use just one approach per test class. I guess it's a convention with the benefit of saving one unecessary import. But maybe I'm making it up 😄.

Anyway, I can certainly adjust it if necessary. 😉.

Reply:
EnabledOn would be more future proof as several *nix flavors are separated out.

Reply:
Fair enough - addressed in 9f541810e43885f1b703152ed67f74ea86ef6e95.

If it makes sense, I can prepare a follow up in order to make this approach consistent all over the testsuite.

Thanks!

Reply:
If you see that in more places it seems good to simplify.